app/models/mention_mailer.rb

```diff
@@ -8,8 +9,9 @@ def self.default_url_options
   
   def notify_mentioning(issue, mentioned_by, mentioned_text, mentioned_user)
     @issue = issue
-    @mentioned_by = mentioned_by
+    @mentioned_by = mentioned_user.name
```

Comment:
Oops. You're right! Ignore anyway, I was using and old master and reimplemented a better patch.